From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Wed, 14 Sep 2016 20:34:45 +0200
Subject: Make AX_ macros optional

Not all AX_ macros aren't available in previous versions of
autoconf-archive, making them optional allows backports.
---
 configure.ac | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2b9137b..c8ed359 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,6 +1,8 @@
 
 AC_INIT(cinnamon-desktop, 3.0.2)
-AX_IS_RELEASE([git-directory])
+
+m4_ifdef([AX_IS_RELEASE], [AX_IS_RELEASE([git-directory])])
+
 AC_CONFIG_SRCDIR(libcinnamon-desktop)
 
 AM_INIT_AUTOMAKE([1.11 foreign no-dist-gzip dist-xz tar-ustar subdir-objects])
@@ -30,7 +32,8 @@ IT_PROG_INTLTOOL([0.40.6])
 AC_PROG_CC
 PKG_PROG_PKG_CONFIG
 
-AX_COMPILER_FLAGS([WARN_CFLAGS],[WARN_LDFLAGS])
+m4_ifdef([AX_COMPILER_FLAGS],
+         [AX_COMPILER_FLAGS([WARN_CFLAGS],[WARN_LDFLAGS])])
 
 AC_ARG_ENABLE(deprecation_flags,
               [AC_HELP_STRING([--enable-deprecation-flags],
